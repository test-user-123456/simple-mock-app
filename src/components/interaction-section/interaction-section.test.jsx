import { render, fireEvent, screen, waitFor } from "@testing-library/react";

import InteractionSection from "./interaction-section";

describe("InteractionSection", () => {
  test("should match snapshot", () => {
    const { asFragment } = render(<InteractionSection />);

    expect(asFragment()).toMatchSnapshot();
  });
});
