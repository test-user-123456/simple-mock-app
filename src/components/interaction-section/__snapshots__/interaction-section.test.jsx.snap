// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InteractionSection should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="row"
  >
    <div
      class="col"
    >
      <div
        class="card"
        data-testid="counting-section"
      >
        <div
          class="card-body"
        >
          <p
            class="card-text"
          >
            Beep Boop Click This Button
          </p>
          <button
            class="btn btn-secondary"
            type="button"
          >
            Boop
          </button>
        </div>
        <div
          class="card-footer"
          data-testid="click-count"
          style="text-align: end;"
        >
          You've clicked the button 
          <b>
            0
          </b>
           times
        </div>
      </div>
    </div>
    <div
      class="col"
    >
      <div
        class="card"
        data-testid="text-input-section"
      >
        <div
          class="card-body"
        >
          <p
            class="card-text"
          >
            What does your heart desire?
          </p>
          <input
            class="form-control"
            placeholder="Tell me your secrets"
            type="text"
          />
        </div>
        <div
          class="card-footer"
          data-testid="text-output"
          style="text-align: end;"
        >
          You didn't tell me anything
        </div>
      </div>
    </div>
    <div
      class="col"
    >
      <div
        class="card"
        data-testid="modal-invoke-section"
      >
        <div
          class="card-body"
        >
          <p
            class="card-text"
          >
            Need a smile? Get a cat!
          </p>
          <button
            class="btn btn-secondary"
            type="button"
          >
            Get cat
          </button>
        </div>
        <div
          class="card-footer"
          data-testid="app-version"
          style="text-align: end;"
        >
          I hope you like your cat
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
